source/distributed/fully_distributed_tria_util.cc

```diff
@@ -512,6 +514,97 @@ namespace parallel
 
         return construction_data;
       }
+
+
+
+      template <int dim, int spacedim>
+      ConstructionData<dim, spacedim>
+      create_construction_data_from_triangulation_in_goups(
```

Comment:
For the other PR where you introduce this function: That's a typo. It should be
```suggestion
      create_construction_data_from_triangulation_in_groups(
```

Reply:
I'll update this type once this PR is merged and I can rebase! Thanks!